waffle-compiler/.npmignore

```diff
@@ -0,0 +1,5 @@
+src
```

Comment:
`.gitignore` seems to be missing

Reply:
As per documentation (https://docs.npmjs.com/misc/developers) files starting with `.git` are automatically ignored

Reply:
My understanding is that only ‘.git’ is ignored (not all prefixed)
> By default, the following paths and files are ignored, so there’s no need to add them to .npmignore explicitly:
> (...)
> .git

Reply:
Checked this manually :) The gitignore file is not present in the bundle when using `npm pack --dry-run`